app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -96,9 +99,13 @@ $(function() {
               query.q[parent + '_eq'] = parentId;
             }
 
+            if (per_page) {
+              query.per_page = per_page
```

Comment:
Identifier 'per_page' is not in camel case                       camelcase<br>Missing semicolon                                                semi